packages/block/src/header-from-rpc.ts

```diff
@@ -39,7 +36,7 @@ export function blockHeaderFromRpc(blockParams: any, options?: BlockOptions) {
       coinbase: miner,
       stateRoot,
       transactionsTrie: transactionsRoot,
-      receiptTrie: isTruthy(receiptRoot) ? receiptRoot : receiptsRoot,
+      receiptTrie: receiptsRoot,
```

Comment:
(side note: this would really need a double-look before a quick change)

Reply:
I've done a bit of research on this and I think we should keep it as is. We could perhaps eventually replace all instances by the `_Root` version of these words (since this is what is returned from RPC endpoints, we could never fully get rid of Root), but we have a lot of test data and commonly used constructors that uses `Trie` so this would be quite a breaking change for a limited benefit.

Reply:
I'll add it to the list of breaking changes considered for v7